templates/homev2_wifi.tpl

```diff
@@ -26,6 +26,7 @@
 #include <VEML6070.h>
 #include <SparkFun_SCD30_Arduino_Library.h>
 #include <LTR329.h>
+#include <ArduinoBearSSL.h>
 
```

Comment:
I'm not sure if we also need to include ```#include <ArduinoECCX08.h>```

Reply:
No need as BearSSL includes the library itself: https://github.com/arduino-libraries/ArduinoBearSSL/blob/master/src/BearSSLClient.cpp#L28